### PR TITLE
Update official.md

### PR DESCRIPTION
**Pull request by Tbkhi**
Changes the layout of the Working Groups section and add the Lang team homepage

### CODE REVIEW

**Comment by Tbkhi** [comment]
Bumping that as "not spam".

**Review by glyn — CHANGES_REQUESTED** [review]
The convention, described in the README of this repo, is that each entry should start with a link containing the literal title of the book. So, for example, https://rust-lang.github.io/wg-async/ has title "wg-async" rather than this PR's "Async team working group", even though that is more explanatory. The description of the entry (after "-" is the place to clarify any confusing book titles).

File: src/official.md [hunk 1]
```diff
@@ -22,12 +22,11 @@ Advanced:
...
 Working groups:
-* [Async team working group](https://rust-lang.github.io/wg-async/) - foundations of async I/O
-* [Compiler team working groups](https://rust-lang.github.io/compiler-team/working-groups/) - a list of
-  working groups
-* [Language design team working group](https://lang-team.rust-lang.org/design_notes.html) - information related to the work of the lang team
-* [Lib `std` developers guide](https://std-dev-guide.rust-lang.org/about.html) - information related to the work of the library team
-* [Types team working group](https://rust-lang.github.io/types-team/) - traits implementation improvements
+* [_Compiler team working groups_](https://rust-lang.github.io/compiler-team/working-groups/) - a list of working groups
+* [Standard library developers Guide](https://std-dev-guide.rust-lang.org/about.html)
+* [T-Lang](https://lang-team.rust-lang.org/design_notes.html) - information related to the work of the lang team
```
Comment (glyn) [inline src/official.md:27]: Let's use the proper title:
```suggestion
* [The Rust Language Design Team](https://lang-team.rust-lang.org/design_notes.html) - information related to the work of the lang team
```

File: src/official.md [hunk 2]
```diff
@@ -22,12 +22,11 @@ Advanced:
...
+* [T-Types](https://rust-lang.github.io/types-team/) - traits implementation improvements
```
Comment (glyn) [inline src/official.md:28]: Let's use the proper title:
```suggestion
* [The Types Team](https://rust-lang.github.io/types-team/) - traits implementation improvements
```

File: src/official.md [hunk 3]
```diff
@@ -22,12 +22,11 @@ Advanced:
...
+* [WG-Async](https://rust-lang.github.io/wg-async/) - foundations of async I/O
```
Comment (glyn) [inline src/official.md:29]: Let's use the same case as the title:
```suggestion
* [wg-async](https://rust-lang.github.io/wg-async/) - foundations of async I/O
```

**Pull request merged**